openshift-kube-apiserver/authentication/oauth/tokenauthenticator.go

```diff
@@ -2,8 +2,11 @@ package oauth
 
 import (
 	"context"
+	"crypto/sha256"
```

Comment:
squash with `UPSTREAM: <carry>: oauth-token-authn: switch to sha256~ prefix`

Reply:
yep, i think this is marked in the sheet to be squashed